core/services/keeper/upkeep_executer_test.go

```diff
@@ -266,19 +267,21 @@ func Test_UpkeepExecuter_PerformsUpkeep_Happy(t *testing.T) {
 	})
 
 	t.Run("errors if submission chain not found", func(t *testing.T) {
-		db, _, ethMock, _, _, _, job, jpv2, _, _, ch, orm := setup(t)
+		db, _, ethMock, _, _, _, _, jpv2, _, keyStore, ch, orm := setup(t)
 
+		registry, jb := cltest.MustInsertKeeperRegistry(t, db, orm, keyStore.Eth(), 0, 1, 20)
```

Comment:
This fix basically prevents the test from using the same job that the `setup()` call creates and adds to the upkeep executer. This allows the test to use a job that might not be getting new runs, since that could be the case based on how the EVM chain ID was being setup and the job being shared between 2 different upkeep executers, if that makes sense. 

@FelixFan1992